### ISSUE TITLE
Swagger UI doesn't show response model

### ISSUE THREAD

**Issue by tcmb** [BigstickCarpet/swagger-suite#6]
I read that swagger-suite is currently being re-written, so I'm not sure how relevant this is. But I noticed that the bundled Swagger UI doesn't show the response model, although the `/docs/` page does show it.

Screenshots from the petstore example for illustration. Here the model is shown:

![screen shot 2015-05-03 at 17 02 00](https://cloud.githubusercontent.com/assets/1877592/7445491/b0b6b8d8-f1b6-11e4-951b-9f1137ec8982.png)

And here it's not:

![screen shot 2015-05-03 at 17 02 41](https://cloud.githubusercontent.com/assets/1877592/7445495/c7622d74-f1b6-11e4-9a70-1e6958929b1a.png)

As a hint, this seems to have worked before April 13th, as a colleague of mine recorded a screencast on swagger-suite and in there the response model schema was shown on the `/ui/` page.

**Comment by BigstickCarpet** [comment]
Yeah, the versions of Swagger-Editor and Swagger-UI that are bundled with Swagger Suite are waaay outdated.  In the new version, these will be installed as NPM dependencies, so they'll always be kept up-to-date.

**Comment by tcmb** [comment]
Thanks for the feedback! We're using Swagger-suite for API prototyping and for decoupling the frontend and backend teams, looking forward to the new version!

**Comment by SteveNewhouse** [comment]
Any update on this? I'm running into same issue.

**Comment by BigstickCarpet** [comment]
Unfortunately, the new version is still a couple months from being done.  We're currently working on all of the underlying projects.  Most of them are done or nearly done now, so you can use those projects directly for now if you want.  It just won't all be bundled up into a nice, neat package (Swagger Suite) until they're all finished.

[Swagger Parser v2.5](https://github.com/BigstickCarpet/swagger-parser/tree/master#swagger-parser) is stable and well-tested.  It was written when Swagger 2.0 was still a _draft_, so it's not 100% compliant with the _final_ spec, but it's about 95% compliant.

[Swagger Parser v3.0.0-alpha](https://github.com/BigstickCarpet/swagger-parser/tree/v3.0.0#swagger-parser) is not thoroughly tested yet, but is written to conform to the _final_ Swagger 2.0 spec.

[Swagger Express Middleware v0.x](https://github.com/BigstickCarpet/swagger-express-middleware#swagger-express-middleware) is very stable and well-tested, but it uses Swagger Parser v2.5, so it's not fully compliant with the final Swagger 2.0 spec. The v1.0.0-alpha version uses Swagger Parser v3.0.0-alpha, so it's more compliant, but not well-tested yet.

[Swagger Server v0.x](https://github.com/BigstickCarpet/swagger-server/tree/0.0.X#swagger-server) is the version that currently ships with Swagger Suite.  It's **way outdated**.  

[Swagger Server v1.0.0-alpha](https://github.com/BigstickCarpet/swagger-server#swagger-server) uses Swagger Express Middleware v1.0.0-alpha, so it's much more up-to-date and compliant with the spec, but not as well-tested yet.


I hope all that was informative.  I promise we're working hard on the new stuff, but it's a **lot** of work, and we both have full-time day jobs and girlfriends that need our attention too.  :)
